### PR TITLE
Restructured `Payments` chapter.

### PR DESCRIPTION
**Pull request by Bosch-0**
Closes https://github.com/BitcoinDesign/Guide/issues/596

Re structured the payments pages based on the above issue. I found the payment requests formats page to make more sense as a sub-page for the requesting bitcoin page, what do you think @GBKS? 

1. This is requesting flows.
2. This is the types of requests that can be used.
3. This is how to send to those formats.

I found this order flowed smoother. 

[Preview](https://deploy-preview-625--sad-borg-390916.netlify.app/)

### CODE REVIEW

**Comment by GBKS** [comment]
Thanks for addressing this.

Do you think the `Payment request formats page` is necessary for readers to understand the `Requesting bitcoin` and `Sending bitcoin` pages? If not, then it's complementary and I think it could be well places towards the end of the section, after `Transaction overview´.

Should `Transaction overview´ maybe be `Transaction lifecycle´?

Make sure add `redirect_from` meta info to pages in which you change the `permalink`. I think there can be multiple entries if a page is moved multiple times.

I could also see us create a `Receiving bitcoin` page and split out that content to give it the space it needs so we can cover it in detail (different PR though).

**Comment by Bosch-0** [comment]
> Do you think the Payment request formats page is necessary for readers to understand the Requesting bitcoin and Sending bitcoin pages? 

I think it adds context to both pages and makes it clearer what kind of things are possible, I don't think you need to understand them all to grasp those pages but it gives a clearer picture.

> Should Transaction overview´ maybe be Transaction lifecycle´?

I prefer overview as it essentially covers more than one lifecycle (on-chain + Lightning)

> I could also see us create a Receiving bitcoin page and split out that content to give it the space it needs so we can cover it in detail (different PR though).

Yeah agree, we may want to go more in-depth with LSP stuff here to as that is where they come in to play.

File: guide/glossary/coin-selection.md [hunk 1]
```diff
@@ -2,10 +2,9 @@
 layout: guide
 title: Coin Selection
 description: A primer on how UTXOs are chosen to fund new bitcoin transactions.
-nav_order: 2
-grand_parent: Payments
-parent: Sending bitcoin
-permalink: /guide/payments/send/coin-selection/
+nav_order: 4
+parent: Glossary
+permalink: /guide/glossary/coin-selection/
```
Comment (GBKS) [inline guide/glossary/coin-selection.md:7]: Need a `redirect_from` here if you're changing the URL.

File: guide/payments/payment-request-formats.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
 title: Payment request formats
-description: An introduction to how sending bitcoin works.
+description: An introduction to the different Bitcoin payment request formats.
 nav_order: 1
 grand_parent: Payments
-parent: Sending bitcoin
-has_children: false
-permalink: /guide/payments/send/payment-request-formats/
+parent: Requesting bitcoin
+permalink: /guide/payments/request/payment-request-formats/
```
Comment (GBKS) [inline guide/payments/payment-request-formats.md:8]: Need a `redirect_from` here if you're changing the URL.

**Review by GBKS — CHANGES_REQUESTED** [review]
Looks great now. Just had two nitpicks about adding redirects for pages where the URL changed.

File: guide/payments/introduction.md [hunk 3]
```diff
@@ -9,6 +9,14 @@ main_classes: -no-top-padding
 image: https://bitcoin.design/assets/images/guide/payments/payments-preview.jpg
 ---
 
+<!--
+
+Illustration source
+
+-
```
Comment (sbddesign) [inline guide/payments/introduction.md:16]: Nit -- did you intend to include an illustration source file here? All I see is the surrounding comment tag.

File: guide/payments/introduction.md [hunk 4]
```diff
@@ -58,15 +60,15 @@ The basics on units and currency symbols, how to format them and what options to
 
 ---
 
-### Managing funds (coming soon)
+### [Transactions overview]({{ '/guide/payments/transactions/' | relative_url }})
 
-How to help users organize and manage their transaction data and holdings.
+An overview of Bitcoin payments, common concepts and the transaction lifecycle.
```
Comment (sbddesign) [inline guide/payments/introduction.md:65]:
```suggestion
An overview of Bitcoin payments, common concepts, and the transaction lifecycle.
```
Oxford comma for consistency.

File: guide/payments/payment-request-formats.md [hunk 5]
```diff
@@ -129,11 +128,11 @@ Each request format has its own unique approach to bundle payment information fo
 
 ---
 
-Sending is one side of the process. Let's look at things from the [receiver's perspective](/guide/payments/request).
+Next we will look at how a [sender](/guide/payments/send) pays these payment requests.
```
Comment (sbddesign) [inline guide/payments/payment-request-formats.md:131]:
```suggestion
Next, we will look at how a [sender](/guide/payments/send) pays these payment requests.
```

File: guide/payments/request.md [hunk 6]
```diff
@@ -457,11 +458,11 @@ For in-person payments, showing the confirmation screen to the sender will likel
 
 ---
 
-Let's go, time go dive in a bit deeper with [transaction privacy]({{ "/guide/payments/privacy/" | relative_url }}).
+Next we go over the different types of [payment request formats](/guide/payments/request/payment-request-formats/) available in bitcoin.
```
Comment (sbddesign) [inline guide/payments/request.md:461]:
```suggestion
Next, we go over the different [payment request formats](/guide/payments/request/payment-request-formats/) available in bitcoin.
```
The comma is grammatically necessary. Removing "types of" is merely my opinion. "Types of" changes the meaning of the sentence. We are actually going over _different payment request formats_ in the chapter; "types of" implies we are looking at "categories of payment request formats" and not the payment request formats themselves.

File: guide/payments/send.md [hunk 7]
```diff
@@ -282,19 +281,19 @@ There are situations in which users may want to make more complex adjustments to
...
+Some users may prefer to choose which of their bitcoin (UTXOs to be precise) to send, in order to protect their privacy. More on this topic on the [Coin selection page]({{ '/guide/glossary/coin-selection/' | relative_url }}) page.
 
 **Lightning routing options**
 
 Routing is a probabilistic endeavor. For example, a routing algorithm may identify two routes. The first one has a low fee but is also less likely to succeed. The second route has a higher fee, but is more likely to succeed. Due to the technical complexity and unknowns, there is [ongoing conversation](https://github.com/BitcoinDesign/Guide/issues/585) whether routing options are relevant for users to be aware of and make decisions on.
 
 ---
 
-Sending is one side of the process. Let's look at things from the [requester's perspective](/guide/payments/request).
+Next we go cover [privacy considerations]({{ '/guide/payments/privacy' | relative_url }}) when to consider when making bitcoin payments.
```
Comment (sbddesign) [inline guide/payments/send.md:292]:
```suggestion
Next, we cover [privacy considerations]({{ '/guide/payments/privacy' | relative_url }}) when to consider when making bitcoin payments.
```
Comma is necessary. "Go" sounds redundant.

File: guide/payments/transactions.md [hunk 8]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: guide
-title: Transactions
-description: An overview of Bitcoin payments, common concepts and the transaction lifecycle.
-nav_order: 1
+title: Transactions overview
+description: An overview of Bitcoin transaction, common concepts and the transaction lifecycle.
```
Comment (sbddesign) [inline guide/payments/transactions.md:4]:
```suggestion
description: An overview of Bitcoin transactions, common concepts, and the transaction lifecycle.
```

File: guide/payments/units-and-symbols.md [hunk 9]
```diff
@@ -203,17 +203,17 @@ Trailing zeros can help more easily compare amounts in right-aligned tables.
...
 
-Complex use cases often require users to interact with addresses, public keys, and other encrypted data, where it is important that each character is easily identifyable. Slashed zeros can help distinguish 0 (the number zero) and O (the uppercase letter "O"), which can look very similar in some typefaces.
+Complex use cases often require users to interact with addresses, public keys, and other encrypted data, where it is important that each character is easily identifiable. Slashed zeros can help distinguish 0 (the number zero) and O (the uppercase letter "O"), which can look very similar in some typefaces.
 
 Note that uppercase letter "O", uppercase letter "I", lowercase letter "l", and the number "0" are not valid characters in addresses, in order to prevent mistakes. As users may not be aware of this, using slashed zeros may still be a helpful design choice.
 
 ---
 
-The next section explores common product designs via [case studies]({{ '/guide/case-studies/' | relative_url }}).
+Next we to a technical deep dive into out [transactions]({{ '/guide/payments/transaction' | relative_url }}) on bitcoin work.
```
Comment (sbddesign) [inline guide/payments/units-and-symbols.md:212]:
```suggestion
Next, we do a technical deep dive into how [transactions]({{ '/guide/payments/transaction' | relative_url }}) on bitcoin work.
```

File: guide/payments/units-and-symbols.md [hunk 10]
```diff
@@ -203,17 +203,17 @@ Trailing zeros can help more easily compare amounts in right-aligned tables.
...
 
 {% include next-previous.html
    previousUrl = "/guide/payments/privacy"
    previousName = "Transaction privacy"
-   nextUrl = "/guide/case-studies/"
-   nextName = "Case studies"
+   nextUrl = "/guide/Payments/transactions"
+   nextName = "Transaction overview"
```
Comment (sbddesign) [inline guide/payments/units-and-symbols.md:218]:
```suggestion
   nextName = "Transactions overview"
```

**Review by sbddesign — CHANGES_REQUESTED** [review]
Thanks, looking good. I don't think I closely followed previous discussions about re-organizing the payments section, but I have no strong opinions about it. The new structure seems reasonable to me.

I left several grammatical, stylistic, and typographical edits. As usual, some of them are grammatically necessary and some are just my opinion on writing style.

File: guide/payments/introduction.md (hunk 3 above)
Comment (Bosch-0) [reply guide/payments/introduction.md:16]: I just got in a habit of adding that there, not sure where the file is for that header @GBKS did you do this one?

Comment (GBKS) [reply guide/payments/introduction.md:16]: @johnsBeharry did that one.

**Review by GBKS — APPROVED** [review]
ACK. Ready to go. Thank you for working through this.

**Review by pavlenex — APPROVED** [review]

**Comment by pavlenex** [comment]
@sbddesign Requires your approval so we can merge it in.

**Review by sbddesign — APPROVED** [review]
Thanks for the edits, LGTM! 💯

**Pull request merged**